requirements-test.txt

```diff
@@ -1,7 +1,7 @@
 coveralls
 responses
 freezegun
-openwisp-users~=0.4.0
+openwisp-users~=0.4.1
```

Comment:
This is unnecessary. Read [compatible release section of PEP 440](https://www.python.org/dev/peps/pep-0440/#compatible-release) specification to know more